tests/integration/targets/inventory/run_ansible_both_false_tests.yml

```diff
@@ -6,6 +6,7 @@
     SC_HOST: "{{ sc_host }}"
     SC_USERNAME: "{{ sc_config[sc_host].sc_username }}"
     SC_PASSWORD: "{{ sc_config[sc_host].sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
```

Comment:
Hm, I know it is not needed, but it should be tested because this env var is used in inventory code. If this was present from the beginning the bug would have been discovered in advance.